package.json

```
@@ -49,7 +49,7 @@
     "postinstall": "npm run bootstrap",
     "preinstall": "npx ensure-node-env",
     "prettier": "prettier --config .prettierrc --write \"**/*.js\"",
-    "publish": "npm run check-pristine && npm run check-owners && npm run build && git pull --rebase && flow stop && npm run test && lerna publish && npm run publish:css",
+    "publish": "npm run check-pristine && npm run check-owners && npm run build && git pull --rebase && flow stop && npm run test && lerna publish && npm run publish:css && ./scripts/publish/process/update-docs-deps.sh",
```

npm has a built-in `postpublish` script that runs automatically after `publish` completes, could we use that for this instead? It'll just help the command from getting too long, and make more sense semantically.

```suggestion
    "publish": "npm run check-pristine && npm run check-owners && npm run build && git pull --rebase && flow stop && npm run test && lerna publish && npm run publish:css",
    "postpublish": "./scripts/publish/process/update-docs-deps.sh",
```

---

Let me take a look at this and see what happens - now I saw it in the npm docs will give it a go!